### PR TITLE
Kobold Monkeys

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

As per this message by Flare, i decided to add kobold monkeys to the game. Kind of like normal monkeys, you can buy them in cubes from cargo and they act similarly. The biggest difference is that while they have a lot less health, they move much faster (i havent balanced movement speed yet) and do more damage than a normal reptilian. When they become sentient, they speak like normal monkeys. If possible, i would like to implement an accent for the kobolds because i cant figure out how to remove the monkey accent/implement normal lizard speech.

image

I haven't implemented a mob like this before and may have made some mistakes in the yamls so please tell me if i did.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image
image

**Changelog**

:cl: Ubaser
- add: You can now purchase kobolds from cargo as an alternative to monkeys!

### CODE REVIEW

**Review by metalgearsloth — DISMISSED** [review]

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 1]
```diff
@@ -1143,6 +1143,132 @@
...
+  - type: GhostRole
+    prob: 0.1
+    makeSentient: true
+    name: ghost-role-information-kobold-name
+    description: ghost-role-information-kobold-description
+  - type: GhostTakeoverAvailable
+  - type: IdExaminable
+  - type: AlwaysRevolutionaryConvertible
+
 - type: entity
```
[the remark this answers is not in the document]
Comment (UbaserB) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:1272]: i have no idea how to do that, i thought you had to list every component

**Comment by UbaserB** [comment]
> These needed to be shaded in the same way as lizards, they currently appear to use many less colors then them.

ok yea kinda forgot about that

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:1272]: That would be better, anything that's just wholesale copying the yaml over as it quickly makes it unmaintainable.

**Comment by Kadeo64** [comment]
can't wait to make an army of kobolds to overthrow the captain

**Comment by BYONDFuckery** [comment]
Pay them each one (1) shiny coin from the vault per shift

**Comment by BYONDFuckery** [comment]
Also make them revvable (with a rare chance to be sentient like monkeys) so you CAN make a kobold army to overthrow the captain


(This might fuck up balancing because SOMEONE is gonna play Emag Kobold and order 20 crates of Kobold Cubes i bet)

**Comment by Errant-4** [comment]
Not gonna lie, "purchase kobolds" just sounds wrong

**Review by Errant-4 — CHANGES_REQUESTED** [review]
The kobolds still speak monkey accent. 

Also can you please merge the main branch because this version of RT will no longer compile on systems so can't be tested

**Pull request merged**